src/containers/BlogCards/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import SingleBlogCard from "../../components/SingleBlogCard";
+import SingleColCard from "../../components/SingleColCard";
+import LoadingPage from "../../components/LoadingPage";
+import "./index.css";
+
+const Stories = () => {
+  const [posts, setPosts] = React.useState([]);
+  const _URL =
+    "https://public-api.wordpress.com/wp/v2/sites/feedtheneedy518055725.wordpress.com/posts";
+
+  React.useEffect(() => {
+    const fetchPosts = async () => {
+      const data = await fetch(_URL);
+      const fetchedPosts = await data.json();
+      setPosts(fetchedPosts);
+    };
+    fetchPosts();
+  }, []);
+
+  if (!posts) {
+    return <LoadingPage />;
+  } else {
+    return (
+      <>
+        <div className="titleLifeContainer">
+          <h1 className="titleLifes">Lifes that changed by you</h1>
```

Comment:
Can you change the color of this header to black? It's inconsistent with other pages headers

Reply:
on the design we made it like this so should I change it as black @cuygur @ihab-sensei @bertacsevercan ?

Reply:
Since the rest of the website has black titles, blue stands out. Let's try to make it consistent.

Reply:
I changed them as black.